{{cookiecutter.project_slug}}/README.rst

```diff
@@ -48,9 +52,15 @@ Initial Setup (APPUiO + GitLab)
 
    .. code-block:: console
 
-        $ oc -n {{ cookiecutter.project_slug }}-production create sa gitlab-ci
-        $ oc -n {{ cookiecutter.project_slug }}-production policy add-role-to-user edit -z gitlab-ci
-        $ oc -n {{ cookiecutter.project_slug }}-production sa get-token gitlab-ci
+{% if cookiecutter.environment_strategy == 'dedicated' -%}
+        oc -n {{ cookiecutter.project_slug }}-production create sa gitlab-ci
+        oc -n {{ cookiecutter.project_slug }}-production policy add-role-to-user edit -z gitlab-ci
+        oc -n {{ cookiecutter.project_slug }}-production sa get-token gitlab-ci
+{% else -%}
+        oc -n {{ cookiecutter.project_slug }} create sa gitlab-ci
+        oc -n {{ cookiecutter.project_slug }} policy add-role-to-user edit -z gitlab-ci
+        oc -n {{ cookiecutter.project_slug }} sa get-token gitlab-ci
+{% endif -%}
```

Comment:
Same here. The solution I found to generate the correct indention and blank lines looks like:
```Jinja2
   .. code-block:: console
{% if cookiecutter.environment_strategy == 'dedicated' %}
        oc -n {{ cookiecutter.project_slug }}-production create sa gitlab-ci
        oc -n {{ cookiecutter.project_slug }}-production policy add-role-to-user edit -z gitlab-ci
        oc -n {{ cookiecutter.project_slug }}-production sa get-token gitlab-ci
{%- else %}
        oc -n {{ cookiecutter.project_slug }} create sa gitlab-ci
        oc -n {{ cookiecutter.project_slug }} policy add-role-to-user edit -z gitlab-ci
        oc -n {{ cookiecutter.project_slug }} sa get-token gitlab-ci
{%- endif %}
```
(Note that there is no blank line after the `.. code-block` statement.)